lib/_http_outgoing.js

```diff
@@ -847,15 +836,13 @@ OutgoingMessage.prototype._flushOutput = function _flushOutput(socket) {
     socket.cork();
   }
 
-  const outputLength = this.outputData.length;
-  if (outputLength <= 0)
+  if (this.outputData.length <= 0)
     return undefined;
 
-  const outputData = this.outputData;
   socket.cork();
   let ret;
-  for (var i = 0; i < outputLength; i++) {
-    const { data, encoding, callback } = outputData[i];
+  for (const outputDataItem of this.outputData) {
+    const { data, encoding, callback } = outputDataItem;
```

Comment:
Why not destructure directly as part of the loop? :)

Reply:
Done in https://github.com/nodejs/node/pull/30958/commits/c5c391602db21045dc7b483d54f0ee91bffa1559! :)